app/_hub/kong-inc/ldap-auth-advanced/index.md

```diff
@@ -78,85 +82,99 @@ params:
       required: true
       default:
       value_in_examples: dc=example,dc=com
+      datatype: string
       description: |
         Base DN as the starting point for the search; e.g., "dc=example,dc=com".
     - name: verify_ldap_host
       required: true
       default: "`false`"
       value_in_examples: false
+      datatype: boolean
       description: |
         Set to `true` to authenticate LDAP server. The server certificate will be verified according to the CA certificates specified by the `lua_ssl_trusted_certificate` directive.
     - name: attribute
       required: true
       default:
       value_in_examples: cn
+      datatype: string
       description: |
         Attribute to be used to search the user; e.g., "cn".
     - name: cache_ttl
       required: true
       default: "`60`"
       value_in_examples: 60
+      datatype: number
       description: |
         Cache expiry time in seconds.
     - name: timeout
       required: false
       default: "`10000`"
       value_in_examples:
+      datatype: number
       description: |
         An optional timeout in milliseconds when waiting for connection with LDAP server.
     - name: keepalive
       required: false
       default: "`10000`"
```

Comment:
the default keepalive is not 10000, it's 60000 instead: https://github.com/Kong/kong-plugin-enterprise-ldap-auth/blob/master/kong/plugins/ldap-auth-advanced/schema.lua#L91

Reply:
Done